internal/html/scrape/schema/json-ld/recipe_test.go

```diff
@@ -21,6 +21,7 @@ func TestRecipeProcessor_GetRecipeNode(t *testing.T) {
 		{name: "parses graph", file: "json-ld-schema-graph.html"},
 		{name: "parses graph with no schema", file: "json-ld-schema-graph-no-schema.html"},
 		{name: "parses node", file: "json-ld-schema-node.html"},
+		{name: "parses graph with array and whitespace", file: "json-ld-schema-as-array-type-array-whitespace.html"},
```

Comment:
This is the test I have added, which is failing at the moment. But again I am unsure how you want this to be handled. :)

Reply:
Instead of checking equality, you could check that `data["type"]` contains the value `Recipe`.